nikola/data/themes/base/templates/comments_helper_utterances.tmpl

```diff
@@ -0,0 +1,26 @@
+## -*- coding: utf-8 -*-
+<%def name="comment_form(url, title, identifier)">
+    %if comment_system_id:
+        <div data-title="${title|h}" id="utterances-thread"></div>
+        <script src="https://utteranc.es/client.js" repo="${comment_system_id}"
+        % if utterances_config:
+        % for k, v in utterances_config.items():
+        ${k}="${v}"
+        % endfor
+        % endif
+        ></script>
+    %endif
+</%def>
+
+<%def name="comment_link(link, identifier)">
+    %if comment_system_id:
+        <a href="${link}#utterances-thread">${messages("Comments")}</a>
+    %endif
+</%def>
+
+
+<%def name="comment_link_script()">
+    %if comment_system_id and 'index' in pagekind:
+        <script src="${comment_system_id}js/count.min.js" data-utterances="${comment_system_id}" data-utterances-lang="${lang}"></script>
```

Comment:
This looks like an isso leftover, right? If uterrances doesn’t have a comment-link thing (that produces "%d comments", like Disqus does), leave this function empty.

Reply:
Oh right. I missed this. I'll have to see if it supports that.

Reply:
I guess it's a requested feature, but not yet supported https://github.com/utterance/utterances/issues/36. So I'll leave this out for now. If there are any updates on that issue we can return to this.